modules/observe/adoption-insights/proc-viewing-total-number-of-users.adoc

```diff
@@ -0,0 +1,15 @@
+:_mod-docs-content-type: PROCEDURE
+[id="proc-total-number-of-users_{context}"]
+= Viewing the total number of users
+
+This card displays the total number of users that have license to use {product}. It also provides a comparison of the number of *Logged-in users* and *Licensed users* in numeric and percentage form. 
+
+* *Logged-in users*: Total number of users, including licensed and unlicensed users, currently logged in to {product-short}.
+
+* *Licensed users*: Total number of licensed users logged in to {product-short}. You can set the number of licensed users in your {product-short} app-config.yaml file.
```

Comment:
You can set the *target* for the number of licenced users...

Does that work better?